clean_hki_accidents.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+
+# File hki_accidents.csv contains all traffic accidents in Helsinki area from 2011 to 2016.
+# It has all the columns of the original dataset except Oslakpvm and Luovpvm.
+
+
+def drop_column(col):
```

Comment:
Either put all the actions mentioned here inside functions, or move useful, often used functions to own files for reusability.

Main function might be useful too . I'd like to avoid accidental data parsing if I'm importing something from this file :)